src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -43,8 +47,15 @@
   private final StringProperty keyServerHttpPort = new SimpleStringProperty("800");
   //TODO: Replace when the StunFileGeneration is ready
   private final String pathToStunDumpFile = this.getClass()
-          .getClassLoader().getResource(STUN_DUMP_FILE_NAME)
-          .toExternalForm().replace("file:","");
+      .getClassLoader().getResource(STUN_DUMP_FILE_NAME)
+      .toExternalForm().replace("file:", "");
+
+
+  private final BooleanProperty vncOptionViewonly = new SimpleBooleanProperty(false);
+  private final BooleanProperty vncOptionWindow = new SimpleBooleanProperty(false);
+  //hard to implement in UI
```

Comment:
I agree but I think we can leave out this comment, I think it makes sense to have it in the model